docs/4.2/enterprise/sso/oidc.md

```diff
@@ -12,7 +12,7 @@ administrators to define policies like:
 !!! warning "Version Warning"
 
     This guide requires a commercial edition of Teleport. The open source
```

Comment:
As a general comment, we should probably change this to "an Enterprise edition of Teleport" to be less confusing. I know we technically have both Enterprise and Pro but we refer to the two collectively as "Teleport Enterprise" everywhere and we say "requires an Enterprise edition of Teleport" elsewhere in some of the guides.

Reply:
Yeah, I agree. Also changed it to be the the Community Edition.